src/Framework/Constraint/Exception.php

```diff
@@ -54,7 +54,8 @@ protected function failureDescription($other)
     {
         if ($other !== null) {
             $message = '';
-            if ($other instanceof Exception) {
+            $throwable = 'Throwable';
```

Comment:
why set the variable first? php doesn't crash if you just write `if ($other instanceof Exception || $other instanceof Throwable) {`